docs/current_docs/manuals/user/cli/354122-services.mdx

```diff
@@ -0,0 +1,45 @@
+---
+slug: /manuals/user/cli/354122/services
+---
+
+# Work with Services
+
+Dagger Functions enable users to spin up long-running services (as containers) and communicate with those services from other Dagger Functions or from the calling host.
+
+Services instantiated by a Dagger Function run in service containers, which have the following characteristics:
+
+- Each service container has a canonical, content-addressed hostname and an optional set of exposed ports.
+- Service containers are started just-in-time, de-duplicated, and stopped when no longer needed.
+- Service containers are health checked prior to running clients.
+
+Some common scenarios for using services with Dagger Functions are:
+
+- Starting a database service in a function and using it for storage or testing
+- Instantiating a service in a function against which end-to-end integration tests can run
+- Running sidecar services
+
+## Start a service returned by a function
```

Comment:
Seems worth mention the other feature of passing host services to functions here (i.e. with the `tcp://host:port`) too.

Not a blocker, could be done when reconciling this with the other Services doc PR